src/shared/Badges/Badges.css

```diff
@@ -23,6 +23,15 @@
 .badge__recruitment {
   background-color: #880e48;
 }
+.badge__recruiter_only {
+  background-color: #880e48;
+}
+.badge__recruiter_both {
+  background-color: #880e48;
+}
+.badge__consultant_only {
```

Comment:
Do we need these classes if the colours are the same?  Should we use `badge__recruitment`?

Reply:
It's because of the `titleMap` object in `Badges.js` - the component is using that to create the different badges and also uses the same value as the class. I figured it was cleaner to just add more classes than change the component to split out the badge's text key and class.